packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -65,7 +89,8 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, Dr
     zIndex: 1000,
     maxHeight: v.listMaxHeight,
     overflowY: 'auto',
-    width: p.fluid ? '100%' : v.width,
+    width: getWidth(p, v),
+    minWidth: 'max-content',
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/52468606-e85a6000-2b91-11e9-94f3-1bc1c25ac11c.png)

https://developer.mozilla.org/en-US/docs/Web/CSS/min-width
This thing is not support in Edge & IE at all 💣

Reply:
damn it..**I saw it used in SUIR though for the same thing**; here are the screenshots for my scenario:

### 1. with `minWidth: 'max-content'`:
![screenshot 2019-02-08 at 12 43 14](https://user-images.githubusercontent.com/5442794/52476555-3f1a6680-2b9f-11e9-80ac-d5c79874edf9.png)

### 2. without `minWidth: 'max-content'`:
![screenshot 2019-02-08 at 12 43 25](https://user-images.githubusercontent.com/5442794/52476562-43468400-2b9f-11e9-8710-e72c3b963f79.png)

Any idea how to achieve this?
@layershifter

Reply:
Actually, I checked SUI code again, they use `white-space: nowrap;` on `item`.

### Before in Edge

![image](https://user-images.githubusercontent.com/14183168/52477658-637c4000-2bab-11e9-8d58-75e8cc20a65a.png)

### After in Edge

![image](https://user-images.githubusercontent.com/14183168/52477672-72fb8900-2bab-11e9-8458-47b64c1bbe39.png)

---

```js
    [`&.${ListItem.className}`]: {
      backgroundColor: v.listItemBackgroundColor,
      whiteSpace: 'nowrap',
    },
```

Reply:
works like a charm, thank you sir 💯